config/initializers/date_and_time_formats.rb

```diff
@@ -2,4 +2,4 @@
 
 # Be sure to restart your server when you modify this file.
 
-Time::DATE_FORMATS[:us] = '%Y/%m/%d %I:%M'
+Time::DATE_FORMATS[:us] = '%Y/%m/%d %H:%M'
```

Comment:
While you're at it, I think d/m/y is more readable - what do you think? Maybe it's not obvious unless you're british though?

Reply:
At the moment time needs to be that structure so that the tables in traction sort correctly by created_at because bootstrap vue does not sort dates. I think its under issue 290 in traction ui issues 👍